### PR TITLE
Added the point about deleting permission rules for categories

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added the point about deleting permission rules for categories

## Affected documentation pages

https://docs.magento.com/user-guide/catalog/catalog-shared-category-permissions-update.html

### CODE REVIEW

File: src/catalog/catalog-shared-category-permissions-update.md [hunk 1]
```diff
@@ -26,5 +26,6 @@ title: Updating Category Permissions
       _Category Permissions Rule_
 
     - To create a new permissions rule for another customer group, click <span class="btn">New Permissions</span> and repeat the process.
+    - To delete permission rule, click the ![]({% link images/images/btn-trashcan2.png %}) icon.
```
Comment (leslietilling) [inline src/catalog/catalog-shared-category-permissions-update.md:29]:
```suggestion
    - To delete a permission rule, click the ![Trash can]({% link images/images/btn-trashcan2.png %}) icon.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
minor edit

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**